rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -940,6 +941,31 @@ void saveNextFederationCreationBlockHeight_postIrisAndHeightIsNotNull_shouldSave
 
     }
 
+    @Test
+    void saveLastRetiredFederationP2SHScript_postIris_getsValueFromStorage() {
+
+        // Arrange
+
+        ActivationConfig.ForBlock irisActivations = ActivationConfigsForTest.iris300().forBlock(0L);
+
+        Script expectedScript = new P2shErpFederationBuilder().build().getP2SHScript();
+
+        StorageAccessor storageAccessor = new InMemoryStorage();
+        FederationStorageProvider federationStorageProvider = new FederationStorageProviderImpl(storageAccessor);
+
+        // Act
+
+        federationStorageProvider.setLastRetiredFederationP2SHScript(expectedScript);
```

Comment:
This line doesn't belong to `Act` part. WDYT if we move it to the `Arrange` part?

Reply:
Updated.